rele/management/commands/document.py

```diff
@@ -0,0 +1,30 @@
+from django.apps import apps
```

Comment:
Management command names are in a global namespace. `python manage.py document` is too generic in the django-wide context. It could be called something like `showsubscriptions`, which also matches the pattern other commands follow, like `showmigrations`